packages/subgraph/subgraph.template.yaml

```diff
@@ -1,4 +1,4 @@
-specVersion: 0.0.4
+specVersion: 0.0.6
```

Comment:
To get all the logs of a transaction. More details [here](https://github.com/graphprotocol/graph-node/pull/3373)